Bio/Align/__init__.py

```diff
@@ -28,6 +28,7 @@
     new_exc.__cause__ = None
     raise new_exc
 
+from Bio.Align import _aligners_old
```

Comment:
I don't see this code - is this as the Python classes using it temporary debugging code?